### ISSUE TITLE
New language from scratch

### ISSUE THREAD

**Issue by codevladimir** [rhasspy/piper#452]
Hello everyone,

I've recently begun experimenting with TTS, and in order to learn more about it, I'm eager to incorporate my native language from the ground up. From what I understand, having a substantial amount of data sets is crucial for achieving optimal results. While starting with 7-8 hours would be a good foundation, aiming for around 24 hours seems ideal (data sourced from LJ Speech, please correct me if I'm mistaken).

Before delving into using an actor's voice, I'm considering using my own voice for a preliminary test to gauge its effectiveness. Would recording around 100 sentences be sufficient for this purpose, with the expectation of exporting a single word regardless of quality (whether it sounds robotic or realistic)? If not, what would you recommend as the minimal data set required to generate one word that's included in the data set?

I understand that it's generally preferable to train models based on existing ones, but since my native language isn't currently supported, I'm opting to start from scratch.

Thank you for your insights.

**Comment by luiscarlos2000** [comment]
Not the developer I am, but just a question: hat new language do you want to be supported on Piper? I want to have more info about it. It's a very interesting project I see.

**Comment by TheStigh** [comment]
Hi @codevladimir Did you ever get to test? I do also want to train my native language from scratch as the only option is a synthesis which is terrible bad (Norwegian). I have about 13 hours of studio voice available, but the most of the information available are for cloning a voice using English with English voice based finetuning based on English voice checkpoints.

**Comment by codevladimir** [comment]
@TheStigh yes i did. I managed to do the simple tests. The sound would be robotic, but that's mostly because my dataset wasn't too big, plus my laptop is underpowered. I had to train it with cpu, since i have amd gpu which is not supported. So i gave up from making it useful
